cli/cmd/plugin/unmanaged-cluster/test/e2e/utils/cli.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2022 VMware Tanzu Community Edition contributors. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package e2e
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/onsi/ginkgo"
+)
+
+func InstallTCE() error {
+
+	wd, err := os.Getwd()
+	if err != nil {
+		log.Println("error while getting current working directory", err)
+	}
+
+	indTCE := strings.LastIndex(wd, "community-edition")
+	topDirPath := wd[0 : indTCE+17]
+	err = os.Chdir(topDirPath)
+	if err != nil {
+		log.Println("error while changing directory to the top:", err)
+		return err
+	}
+
+	// installing dependencies i.e docker and kubectl
+	err = runDeployScript("hack/ensure-deps/check-docker.sh")
+	if err != nil {
+		log.Fatal(err)
+		return err
+	}
+
+	err = runDeployScript("hack/ensure-deps/ensure-kubectl.sh")
+	if err != nil {
+		log.Fatal(err)
+		return err
+	}
+
+	// build.sh is creating a release and installing TCE for latest code
+	return runDeployScript("test/build-tce.sh")
+}
+
+func runDeployScript(filename string) error {
+	mwriter := io.MultiWriter(os.Stdout)
+	cmd := exec.Command("/bin/bash", filename)
+	cmd.Stderr = mwriter
+	cmd.Stdout = mwriter
+	err := cmd.Run() // blocks until sub process is complete
+	if err != nil {
+		log.Fatal(err)
+	}
+	return err
+}
+
+func Kubectl(input io.Reader, args ...string) (string, error) {
+	return cliRunner("kubectl", input, args...)
+}
+
+func Tanzu(input io.Reader, args ...string) (string, error) {
+	return cliRunner("tanzu", input, args...)
+}
+
+func cliRunner(name string, input io.Reader, args ...string) (string, error) {
+	fmt.Fprintf(ginkgo.GinkgoWriter, "+ %s %s\n", name, strings.Join(args, " "))
```

Comment:
If we won't be using the Ginkgo / Gomega testing suite, can we remove use of the `GinkgoWriter` so we can remove the Ginkgo dependency all together?